neofetch

```diff
@@ -2884,20 +2885,26 @@ kde_config_dir() {
     if [[ "$kde_config_dir" ]]; then
         return
 
-    elif [[ -n "$KDE_CONFIG_DIR" ]]; then
-        kde_config_dir="$KDE_CONFIG_DIR"
+#    elif [[ -n "$KDE_CONFIG_DIR" ]]; then
+#        kde_config_dir="$KDE_CONFIG_DIR"
+
+    elif type -p kf5-config >/dev/null 2>&1; then
+        kde_config_dir="$(kf5-config --path config)"
+        kde_config_dir="${kde_config_dir/$'/:'*}"
 
     elif type -p kde4-config >/dev/null 2>&1; then
-        kde_config_dir="$(kde4-config --localprefix)"
+        kde_config_dir="$(kde4-config --path config)"
+        kde_config_dir="${kde_config_dir/$'/:'*}"
 
     elif type -p kde-config >/dev/null 2>&1; then
-        kde_config_dir="$(kde-config --localprefix)"
+        kde_config_dir="$(kde-config --path config)"
+        kde_config_dir="${kde_config_dir/$'/:'*}"
 
     elif [[ -d "${HOME}/.kde4" ]]; then
-        kde_config_dir="${HOME}/.kde4"
+        kde_config_dir="${HOME}/.kde4/share/config"
 
     elif [[ -d "${HOME}/.kde3" ]]; then
-        kde_config_dir="${HOME}/.kde3"
+        kde_config_dir="${HOME}/.kde3/share/config"
     fi
```

Comment:
Can we not just move `kde_config_dir="${kde_config_dir/$'/:'*}"` to the bottom of the function? This way we only have it once instead of 3 times.